app/api/notify-token/notify-token.ts

```diff
@@ -0,0 +1,28 @@
+const formUrl = 'https://docs.google.com/forms/d/e/1FAIpQLSfx0jx3sNnxkJxWiScPiiMVVSQW3EaOUEtkjwv8phhEBKlonw/formResponse'
+const tokenEntryId = 'entry.1169450829'
+const emailEntryId = 'entry.1642066663'
```

Comment:
Please make this API wrapper reusable, we need it for token and chain.
You can make some abstract GoogleFormService class and then I can use it where need